runtime/westend/src/lib.rs

```diff
@@ -1835,3 +1859,46 @@ mod remote_tests {
 		ext.execute_with(|| Runtime::on_runtime_upgrade());
 	}
 }
+
+mod init_state_migration {
+	use super::Runtime;
+	use frame_support::traits::OnRuntimeUpgrade;
+	use pallet_state_trie_migration::{AutoLimits, MigrationLimits, MigrationProcess};
+	#[cfg(not(feature = "std"))]
+	use sp_std::prelude::*;
+
+	/// Initialize an automatic migration process.
+	pub struct InitMigrate;
+	impl OnRuntimeUpgrade for InitMigrate {
```

Comment:
Will you put this into Substrate for future use or always copy&paste?

Reply:
More copy&paste, I mean this is specific to the migration strategy of the chain.